WebRoot/templates/mail/Message.template

```diff
@@ -67,19 +72,30 @@
 </template>
 
 <template id='mail.Message#SentBy'>
-	<span>${sentBy}</span>
 	<$
 		var useObo = data.obo && (data.oboAddr != data.sentByAddr);
 		var useBwo = data.bwo && (data.bwoAddr != data.sentByAddr);
-		if (useObo) {
-			$><span style='margin:0 .5em;'><$= ZmMsg.onBehalfOfMidLabel $></span>
-			<span id="${oboId}"><$= data.obo $></span><$
-		}
-		if (useBwo) {
-			$><span style='margin:0 .5em;'><$= ZmMsg.byWayOfMidLabel $></span>
-			<span id="${bwoId}"><$= data.bwo $></span><$
-		}
 	$>
+	<table role="presentation">
+		<tbody>
+			<tr>
+				<td>${sentBy}</td>
+				<$ if (useObo) { $>
+					<td class="InfoLabel">&nbsp;&nbsp;<$= ZmMsg.onBehalfOfMidLabel $>&nbsp;&nbsp;</td>
```

Comment:
Yes, and we should I think. Updated code: https://github.com/Zimbra/zm-web-client/pull/60/commits/194518e417f293c3a004c03198a99d6d148ae29e#diff-0af604f378b187bd0c6e8440df62130eL84